blog/2023-06-12-bringing-dlc-to-lightning-part-1.md

```diff
@@ -0,0 +1,82 @@
+---
+slug: dlc-to-lightning-part-1
+title: Bringing DLC to Lightning Part 1
+authors: [richard]
+tags: [DLC, lightning, knowledge-base]
+---
+
+At 10101, our mission is to provide self-custodial trading solutions for Bitcoin traders, and we have been dedicated to this goal for a significant period of time.
+Our initial solution, known as ItchySats, involved implementing Layer 2 Discreet Log Contracts (DLCs) directly on Layer 1. 
+However, we quickly realized that on-chain transactions were not efficient enough in terms of speed and cost to deliver the trading experience we envisioned.
```

Comment:
This is not 100% correct, we did build our own layer-2 protocol https://github.com/comit-network/maia/.

Reply:
Whats not correct? I am saying that our initial solution implemented a Layer 2.

```
... implementing Layer 2 Discreet Log Contracts (DLCs) directly on Layer 1.
```

Reply:
Reworded the whole paragraph.